BLSQ/iaso
# POLIO-1439 move config model

Config model was in polio, but we need it in iaso for OpenHexa integration

Related JIRA tickets : POLIO-1439

## Self proofreading checklist


## Doc
Added a docstring in the Model

## Changes

- Added a `Config`Model in iaso
- Copied all polio `Config` to iaso's
- Delete poliio `Config`
- Update imports
- Move `ConfigViewSet` to iaso
- Move `Config` admin to iaso admin
- Add /api/configs/ to iaso. The old /api/polio/configs/ was updated but not deleted as it's used by PowerBI dashboards

## How to test

- On a polio DB: apply the migrations, go to admin --> all configs should be there
- Impacted features:
    - IM country view
    - lqas refresh button 
    - preparedness google sheet generation
    - preparedness email warning
    - forma api endpoint
    - powerBi refresh API


## Print screen / video

There should be no visible change

## Notes

⚠️ This can have a big impact on PowerBi dashboard if something goes wrong, we should test it on staging
merges into https://github.com/BLSQ/iaso/pull/1071
